.changeset/shiny-ants-cover.md

```diff
@@ -0,0 +1,12 @@
+---
+'@sl-design-system/combobox': patch
```

Comment:
Can you please also add changeset for other component's changes like grid, listbox, tag and so on? :)